types/is-date-object/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for is-date-object 1.0
+// Project: https://github.com/ljharb/is-date-object
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.3
```

Comment:
Could you also add definitions compatible with older TypeScript versions? You can follow the instructions here: https://github.com/DefinitelyTyped/DefinitelyTyped/#i-want-to-use-features-from-typescript-31-or-above

Reply:
Good point, I've added simpler type defs for older versions of TS but I'm getting some weird compile errors:

```
expect  TypeScript@next compile error: 
Expected 1 arguments, but got 0
```

I've followed the docs and mimicked other TS projects with multiple definitions, is this a simple fix?